manifests/0000_20_etcd-operator_11_clusterrole.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: etcd-operator-cluster-role
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  rules:
+  - apiGroups:
+    - ""
+    resources:
+    - configmaps
+    verbs:
+    - get
+    - list
+    - patch
+    - update
+    - watch
+  - apiGroups:
+    - ""
+    resources:
+    - namespaces
+    - nodes
+    - secrets
+    verbs:
+    - get
+    - list
+    - watch
```

Comment:
I think we're missing a create here to create the openshift-etcd namespace. @Elbehery can you go through all the resources that bootkube must create? They can be found here: https://github.com/openshift/cluster-etcd-operator/tree/master/bindata/bootkube/manifests

Reply:
even though I believe this should happen before the operator runs, so I'll dig deeper on Monday, maybe you can figure it out already from the install bundles :)

Reply:
SGTM .. it will be done before Monday ;)